docs/BUILD.md

```diff
@@ -1,27 +1,29 @@
 # Building with HMT
 
 HMT builds OpenAPI specs from two possible sources:
-- Recordings of HTTP API traffic serialized in the [`http-types`](https://github.com/hmt/http-types) format and written to a [`.jsonl`](https://jsonlines.org) file
+- Recordings of HTTP API traffic serialized in the [`http-types`](https://github.com/Meeshkan/http-types) format and written to a [`.jsonl`](http://jsonlines.org) file
 - Other OpenAPI specs
 
 In case one only has an OpenAPI spec and no server recodings, it is not necessary to run HMT build and you can go directly to the [`hmt mock`](./MOCK.md) command.
 
 ## What's in this document
 
-- [The `hmt build` command](#the-hmt-build-command)
+- [Building with HMT](#building-with-hmt)
+  - [What's in this document](#whats-in-this-document)
```

Comment:
```suggestion

```

The page title and Table of Contents title don't need to be part of the actual Table of Contents. So I'd recommend removing and then tweaking the indentation of the rest of the list if that causes issues.